src/net/if_.rs

```diff
@@ -26,6 +26,22 @@ pub fn if_nametoindex<P: ?Sized + NixPath>(name: &P) -> Result<c_uint> {
     }
 }
 
+/// Resolve an interface number into an interface.
+pub fn if_indextoname(index: c_uint) -> Result<CString> {
+    // We need to allocate this anyway, so doing it directly is faster.
+    let mut buf = vec![0u8; IF_NAMESIZE];
+
+    let return_buf = unsafe {
+        libc::if_indextoname(index, buf.as_mut_ptr() as *mut _)
```

Comment:
For pointer casting, [we prefer to use `.cast()` over the `as` keyword](https://github.com/nix-rust/nix/blob/master/CONVENTIONS.md#pointer-type-casting):

```suggestion
        libc::if_indextoname(index, buf.as_mut_ptr().cast())
```

Reply:
Ok, I'm going to fix this later today.